src/witan/sen2/return/person_level/blade/eda.clj

```diff
@@ -44,8 +44,8 @@
   [ds cols]
   (reduce 
    (fn [m k] 
-     (merge m 
-            {k (frequencies (ds k))}))
+     (merge m
+            (when (ds k) {k (frequencies (ds k))})))
```

Comment:
How come `when` here?

Reply:
Added the when to avoid summarising columns that are not in the dataset. Wasn't an issue before but now code deals with both v1.1 & v1.2 COLLECT exports there are some fields in one but not the other.